superset/db_engine_specs.py

```diff
@@ -866,6 +866,18 @@ def convert_dttm(cls, target_type, dttm):
 
 class RedshiftEngineSpec(PostgresEngineSpec):
     engine = 'redshift'
+    
+    time_grains = (
```

Comment:
isn't this inherited from `PostgresEngineSpec`? Looks like a copy-paste of exactly what is inherited...